rocAL/rocAL/include/mxnet_recordio_reader.h

```diff
@@ -0,0 +1,111 @@
+/*
+Copyright (c) 2019 - 2020 Advanced Micro Devices, Inc. All rights reserved.
```

Comment:
modify the date to 2022 applicable to all files changed in this PR

Reply:
Done